clang/test/SemaSYCL/intel-fpga-local.cpp

```diff
@@ -670,20 +347,8 @@ void diagnostics()
   [[intel::force_pow2_depth]] unsigned int force_p2d_no_args[64];
   //expected-error@+1{{'force_pow2_depth' attribute takes one argument}}
   [[intel::force_pow2_depth(0, 1)]] unsigned int force_p2d_2_args[64];
-
-  // Checking of different argument values.
-  //CHECK: VarDecl{{.*}}force_p2d_dup
-  //CHECK: SYCLIntelMemoryAttr{{.*}}Implicit
-  //CHECK: SYCLIntelForcePow2DepthAttr
-  //CHECK-NEXT: ConstantExpr
-  //CHECK-NEXT: value:{{.*}}1
-  //CHECK-NEXT: IntegerLiteral{{.*}}1{{$}}
-  //expected-note@+2{{previous attribute is here}}
-  //expected-warning@+1{{attribute 'force_pow2_depth' is already applied with different arguments}}
```

Comment:
Same here

Reply:
Yes, AST part we are checking inside clang/test/SemaSYCL/intel-fpga-local-ast.cpp file at line 259-267
Diagnostic part we are checking inside clang/test/SemaSYCL/intel-fpga-local.cpp file at line 106-109